### ISSUE TITLE
THE WINDOW IS WAY TOO FAT FOR MY LAPTOP

### ISSUE THREAD

**Issue by peanutbuttery2763** [carykh/VirusGame#37]
THE WINDOW IS **TOO FAT!!!!!** AND I CANT EVEN RESIZE IT OR CHANGE IT IN THE CODE

![image](https://github.com/user-attachments/assets/4089444d-1e44-43dc-b011-ff90e774f576)

**Comment by i3vie** [comment]
yeah, on my 1600x900 display the window way overruns the display and changing W_W and W_H values doesn't help, just seems to break the UI
